app/views/form-designer/page-preview.html

```diff
@@ -45,7 +45,10 @@
           label: label,
           hint: {text: pageData['hint-text']},
           id: pageId,
-          name: pageId
+          name: pageId,
+          inputmode: "numeric",
+          pattern: "[0-9]*",
```

Comment:
We might want to allow `.` and `,` here
```suggestion
          pattern: "[0-9.,]*",
```

Reply:
Ignore that!